internal/route/routes.go

```diff
@@ -0,0 +1,135 @@
+package route
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/gorilla/csrf"
+	"github.com/gorilla/mux"
+	"github.com/microcosm-cc/bluemonday"
+	"gitub.com/imartingraham/todobin/internal/model"
+)
+
+// HandleIndex is the route for "/"
+func HandleIndex(w http.ResponseWriter, r *http.Request) {
+	tpl := template.Must(template.ParseFiles("web/template/index.html"))
```

Comment:
FYI: In a production site you might move the `template.Must(template.Parsefiles(...` outside a handler. You can parse multiple files (see godot) and request the individual template for `tpl.Execute...`.

This would mean templates are compiled once per application start and not on each request. An example of using that shared memory.